aspnet/fundamentals/error-handling/sample/src/ErrorHandlingSample/Startup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Net;
+using System.Text;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.WebEncoders;
+
+namespace ErrorHandlingSample
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit http://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, 
+            IHostingEnvironment env)
+        {
+            app.UseIISPlatformHandler();
+
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            app.UseStatusCodePagesWithReExecute("/errors/{0}");
+
+            app.MapWhen(context => context.Request.Path == "/", HomePage);
```

Comment:
And then put in a Run for HomePage at the end (I also want the homepage behavior to stay)?